package.json

```diff
@@ -5,10 +5,10 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@esbuild/win32-x64": "^0.18.20",
```

Comment:
Is it relevant for this pull request?

Reply:
I needed that to run it locally on my windows pc, until i'd installed that, i couldn't run it locally. so maybe it is a dependency that works fine at my end. the changes introduced has just a file created which handles localStorage as a custom hook.

Reply:
Could you please open an issue regarding the problem you experienced with running the project locally?
If you can, please supply the errors you've seen, and provide additional metadata regarding your environment.
AFAIK, the current configuration works well for windows developers

Reply:
I'm having a hard time reproducing this error though, but unrelated to this, I had this vite-plugin error and I resolved it by installing the latest version of vite:
```npm i -D @vitejs/plugin-react-swc@latest```


<img width="1328" alt="vite-error" src="https://github.com/hasadna/open-bus-map-search/assets/50064240/a2559fe0-7b36-4fe4-b5c0-92476516c8bf">

Reply:
what is node version you r using?

Reply:
v20.10.0 :)

Reply:
what version of windows?

Reply:
Windows 11 Pro, Version 21H2

Reply:
i think it can be good idea to upgrade @vitejs/plugin-react-swc package to latest

Reply:
or may be we can consider to replace it with `@vitejs/plugin-react` (which looks like something similar to me)

Reply:
Yup that fixed it for me, thanks for the suggestion @shootermv. Should I still raise this as an issue?

Reply:
When I ran this command, ```npm i -D @vitejs/plugin-react-swc@latest```, it resolved the issue.

Reply:
so far - i can see lot of `lint` issues in the code (which are small code styling issues)
i think you should run `npm run format` command

Reply:
Had this error on my terminal though
![npm_run_format](https://github.com/hasadna/open-bus-map-search/assets/50064240/6ef2077d-e75a-49a6-ae1c-e64dcc472e7f)

Reply:
I used prettier to format the files that I had changes made in. Is it good enough @shootermv ?

Reply:
sorry for misleading,i guess, the correct command for lint fixing should be `npm run lint:fix`

Reply:
Ran the command which made the necessary corrections. Please let me know if there are any other issues with this. Thanks!

Reply:
please revert `package-lock.json` and `package.json`, seems like it makes the CI fail.
```
git checkout main package.json
git checkout main package-lock.json
```

Reply:
@NoamGaash, Done!